server/pages-helpers.ts

```diff
@@ -72,6 +72,39 @@ const getEntryForPath = (fs, filePath) => {
   };
 };
 
+const sortByTitle = (a, b) => {
```

Comment:
can you add a comment here describing the algorithm chosen for sorting? "if it includes this, sort alphabetically, otherwise go by title" or whatever

Reply:
Added in 52328f072c721e2621bc5acf6d39177e1ae4ccc7